src/VCF_Tasks.cpp

```diff
@@ -44,10 +48,10 @@ bool run_read_adc2_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo&
     adc_2.sample(); // Samples all eight channels.
```

Comment:
we're gonna change ADC implementations elsewhere, but just to get your code to compile, replace `adc_1.sample(); adc_1.convert()` with `adc_1.tick()`.